src/backend/expungeservice/crawler/parsers/case_parser.py

```diff
@@ -108,13 +108,13 @@ def __parse_event_table(event) -> Optional[Dict[int, Dict[str, str]]]:
             else:
                 raise ValueError("len(event_table_contents) should always be 4 or 5.")
 
-            if CaseParser.__normalize_text(event_type.text) == "disposition":
+            if CaseParser.__normalize_text(event_type.text) in ["disposition", "amendeddisposition"]:
                 disposition_data = {}
                 for row, next_row in zip(event_inner_table_parse, event_inner_table_parse[1:]):
                     if CaseParser._valid_data(row.split(".\xa0")):
                         charge_id_string, charge = row.split(".\xa0")
                         charge_id = int(charge_id_string)
-                        disposition_data[charge_id] = {"date": date.text, "charge": charge, "ruling": next_row}
+                        disposition_data[charge_id] = {"date": date.text, "charge": charge, "ruling": next_row, "event": event_type.text}
```

Comment:
Does this rely on the fact that the amended disposition always comes after the disposition entry?

Reply:
Yep. It seems like this reliably happens because the events are listed in chronological order. This approach fixes the existing bug reports for this issue, though we could make something more sophisticated later.